lib/ts/recipe/emailverification/recipe.ts

```diff
@@ -40,11 +42,15 @@ export default class Recipe extends RecipeModule {
 
     isInServerlessEnv: boolean;
 
+    emailDelivery: EmailDeliveryRecipe<TypeEmailVerificationEmailDeliveryInput>;
+
     constructor(recipeId: string, appInfo: NormalisedAppinfo, isInServerlessEnv: boolean, config: TypeInput) {
```

Comment:
Should accept ingredients object (just like emailverification does). And then emailDelivery should be assigned to that if it is provided.